src/openrct2/paint/Supports.h

```diff
@@ -94,6 +94,19 @@ enum class MetalSupportType : uint8_t
     BoxedCoated,
 };
 
+enum class MetalSupportPlace : uint8_t
```

Comment:
```suggestion
/* Tiles are rotated 45 degrees, and divided as shown here:
       0
   5       6
1      4      2
   7       8
       3
*/
enum class MetalSupportPlace : uint8_t
```

What do you think of documenting it here as such?

Reply:
Good idea, I’ll push a commit to add it (and some additional clarification).